components/stream-provider/src/__tests__/browser/application.spec.ts

```diff
@@ -1,6 +1,6 @@
 import pino from "pino";
 import { IInputHandler } from "../../base/input-handler";
-import { BaseSignalProvider } from "../../base/signal-provider";
+import { BaseSignalProvider } from "../../../../shared/src/signal-provider"
```

Comment:
this is how the `bootstrap` command is designed to work/let us access deps?

Reply:
Yes. When I reference by the name of the package. It is never found